src/views/home/index.tsx

```diff
@@ -17,6 +19,9 @@ export const HomeView = () => {
   const SOL = useUserBalance(WRAPPED_SOL_MINT);
   const { balanceInUSD: totalBalanceInUSD } = useUserTotalBalance();
 
+  const [oauthToken, setOAuthToken] = useState<any>(null);
+  const [userInfo, setUserInfo] = useState<any>(null);

```

Comment:
lets not use any

Reply:
Agreed. Although the scaffold seems to settle for it in places. I think in the name of getting this across the finish line maybe we make this a minimization problem haha. Partially done! Lmk your thoughts.

Reply:
I think we should tackle this one now

Reply:
I'm not sure if 1. we're going to want *all* of the user info returned. 2. I also think when/if we use the SDK we will probably abstract this away anyway. I don't mind focusing on this more, but if we do that I'd like to spend more time with octokit.js. What do you think?

Reply:
Hmm I see what you mean, i just realized these things are coming from an external api.
What if we do some type like
```
const userInfoType = {
 name: string; // or something we will always get
 [key: any]: any // to be aware that some key and value will be undetermined
}
```
what do u think?

Reply:
lgtm. seems simple and we can always build onto it.

Reply:
question

what exactly is `[key: any]`? 

```
An index signature parameter type must be 'string', 'number', 'symbol', or a template literal type
```

I noticed you used `const` for this struct. Should i not use `interface`?

Reply:
yea it was kinda a pseudocode, you can use type or interface. i forgot the syntax for making TS types when i wrote that. 😅 

```
type userInfoType = {
  [key: any]: any;
 ... other fields here
  }

Reply:
I just went ahead and added the whole thing. We can remove it when we use `octokit.js`.